workspace.md

```diff
@@ -0,0 +1,98 @@
+- Commits are automatically linted using `commitlint` with a husky git hook.
+- [Commitizen](https://www.npmjs.com/package/commitizen) can easily configured to format commit message
+
+## Semantic Releases
+
+- We automatically release from `master` branch using [semantic-release](https://github.com/semantic-release/semantic-release) and [semantic-release-monorepo](https://github.com/Updater/semantic-release-monorepo)
+- Changes to each `package/` are automatically detected and we only release packages with changes
+- The angular commit messages used to determine if the changes are `patch`, `minor`, or `marjor`
+
+| Commit message                                                                                              | Release type               |
+| ----------------------------------------------------------------------------------------------------------- | -------------------------- |
+| `fix(ticket): fix bug to create patch`                                                                      | Patch Release              |
+| `feat(ticket): add new feature is a minor release`                                                          | ~~Minor~~ Feature Release  |
+| `perf(ticket): remove graphiteWidth option`<br><br>`BREAKING CHANGE: This block will trigger major release` | ~~Major~~ Breaking Release |
+
+#### Default rules matching
+
+If a commit doesn't match any rule in `releaseRules` it will be evaluated against the.
+
+With the previous example:
+
+- Commits with a breaking change will be associated with a `Major` release.
+- Commits with `type` 'feat' will be associated with a `minor` release.
+- Commits with `type` 'fix' will be associated with a `patch` release.
+- Commits with `type` 'perf' will be associated with a `patch` release.
+
+#### No rules matching
+
+If a commit doesn't match any rules in `releaseRules` then no release type will be associated with the commit.
+
+With the previous example:
+
+- Commits with `type` 'style' will not be associated with a release type.
+- Commits with `type` 'test' will not be associated with a release type.
+- Commits with `type` 'chore' will not be associated with a release type.
+
+#### Multiple commits
+
+If there is multiple commits that match one or more rules, the one with the highest release type will determine the global release type.
+
+Considering the following commits:
+
+- `docs(SB-xxx): Add more details to the API docs`
+- `feat(SB-xxx): Add a new method to the public API`
+
+With the previous example the release type determine by the plugin will be `minor`.
+
+### Manual Release Process
+
+The idea here is we need to rebase the changes in `dev` onto the `master` branch. Since we rely on commit messages in our `semantic-release`, we can't apply a merge commit.
+The recommended flow is as follows assuming `master` is in synch with `dev`.
+
+1. Developer opens a `feat|fix` PR
+1. When the PR is approved, we can either `rebase` or `squash and merge` into `dev`
```

Comment:
Do we have a preference for `rebase` vs `squash and merge`?

Reply:
This was originally wrote before some more recent changes to how we actually are working within    Since this looked like a re-organization of what we have I was okay with it, maybe we should add a task to Jira to update these docs to reflect current process.